pkgs/development/tools/misc/sipp/default.nix

```diff
@@ -1,29 +1,30 @@
-{lib, stdenv, fetchurl, ncurses, libpcap }:
+{ lib, stdenv, fetchurl, ncurses, libpcap, cmake, openssl, git }:
 
 stdenv.mkDerivation rec {
-  version = "3.6.0";
+  version = "3.6.1";
 
   pname = "sipp";
 
   src = fetchurl {
-    url = "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
-    sha256 = "1fx1iy2n0m2kr91n1ii30frbscq375k3lqihdgvrqxn0zq8pnzp4";
+    url =
+      "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-alYOg6/5gvMx3byt+zvVMMWJbNW3V91utoITPMhg7LE=";
   };
 
-  postPatch = ''
-    sed -i "s@pcap/\(.*\).pcap@$out/share/pcap/\1.pcap@g" src/scenario.cpp
+  configurePhase = ''
+    cmake . -DUSE_GSL=1 -DUSE_PCAP=1 -DUSE_SSL=1 -DUSE_SCTP=0
   '';
 
-  configureFlags = [
-    "--with-pcap"
-  ];
+  buildPhase = ''
+    make -j4
+  '';
```

Comment:
Overwriting the build phase like that is not required when you have the correct build hooks in place.

Reply:
If I omit buildPhase nix isn't using the provided patched tarball but the git source which lacks the two missing but necessary files to build.
`version.h and sipp.1`

How can I fix this?🤔

Reply:
I think this is more related to the location of the cmake call. The upstream uses `cmake .`, so it creates the build inside the source directory.

Reply:
Okay, the build is just significantly slower. How would I pass that to nix instead?

Reply:
Set `enableParallelBuilding = true`.